src/unpackertool/plugins/upx/unfilter.cpp

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
The [current Windows TeamCity build](https://retdec-tc.avast.com/viewLog.html?buildId=1515&buildTypeId=Retdec_WinBuild&tab=buildLog) fails with a lot of the following errors:
```
[21:43:46] :	 [Step 1/6]     68>e:\work\414c7650a8fb97ac\retdec\src\unpackertool\plugins\upx\unfilter.cpp(88): error C2039: 'min': is not a member of 'std' [E:\work\414c7650a8fb97ac\retdec\build\src\unpackertool\plugins\upx\retdec-unpacker-upx.vcxproj]
[21:43:46] :	 [Step 1/6]     68>e:\work\414c7650a8fb97ac\retdec\src\unpackertool\plugins\upx\unfilter.cpp(130): error C2039: 'min': is not a member of 'std' [E:\work\414c7650a8fb97ac\retdec\build\src\unpackertool\plugins\upx\retdec-unpacker-upx.vcxproj]
...
```
I believe that you are missing `#include <algorithm>` in the file (this is where `std::min()` is defined). Alternatively, if the file transitively includes `windows.h`, we will probably need to compile with [NOMINMAX](https://stackoverflow.com/questions/4913922/possible-problems-with-nominmax-on-visual-c).